js/components/viewControlsVr.js

```diff
@@ -18,15 +18,42 @@ var ViewControlsVr = React.createClass({
       e.cancelBubble = true; // IE
       this.props.controller.state.accessibilityControlsEnabled = true;
     }
+
     this.props.controller.moveVrToDirection(isRotated, direction);
   },
 
+  /**
+   * @method ViewControlsVr#_setIconSymbol
+   * @private
+   */
+  _setIconSymbol: function () {
+    var desktopContent = this.props.skinConfig.buttons.desktopContent;
+    this.icon = _.find(desktopContent, function (el) {
+      return el.location === "mainView";
+    });
+  },
+
+  /**
+   * @method ViewControlsVr#_setSubstrateSymbol
+   * @private
+   */
+  _setSubstrateSymbol: function () {
```

Comment:
Same thing here, change 'set' to something else since I can't set it to a value.  Also what does substrate mean here?  I see that we've used it a couple of times in the VR code but it's confusing what it means.  Is this the VR UI surface?

Reply:
fix